### PR TITLE
Update WindowsOSTrainingGuide.md

### PR DESCRIPTION
**Pull request by ajstinn2**
- Commands and language updated to reflect 3.8.0
- Better linking to downloads page
- Removed passing the APP_ID, environment variable, from being its own step. And included it in-line with the HawkScan command. This is in an effort to decrease the overall commands needed. And keep it as close as possible to the MacOS / Linux training.

### CODE REVIEW

File: WindowsOSTrainingGuide.md [hunk 1]
```diff
@@ -3,19 +3,19 @@
 This guide will be used in tandem with the live training given by the StackHawk Team. It can also be used in a self-paced fashion to walk through the steps of getting started with HawkScan. Throughout the training, we will reference specific commands that must be entered in your terminal. Those commands are outlined below, with detailed descriptions.
 
 ## Step 1: _Installing HawkScan_
-There are multiple different methods for [downloading](https://docs.stackhawk.com/stackhawk-cli/#installation-overview) the HawkScan CLI tool. But for the purpose of this training, we will install HawkScan via our MSI Installer. Run the following command in your terminal to download and install the latest version of HawkScan.
+There are multiple different methods for [downloading](https://docs.stackhawk.com/download.html) the HawkScan CLI tool. But for the purpose of this training, we will install HawkScan via our MSI Installer. Run the following command in your terminal to download and install the latest version of HawkScan.
```
Comment (kcberg) [inline WindowsOSTrainingGuide.md]: There are multiple install methods and [downloads](https://docs.stackhawk.com/download.html) of the HawkScan CLI tool.

**Review by kcberg — CHANGES_REQUESTED** [review]
words

File: WindowsOSTrainingGuide.md (hunk 1 above)
Comment (ajstinn2) [reply WindowsOSTrainingGuide.md]: @kcberg Changed the language to better reflect our install and download methods in latest commit

**Review by azconger — DISMISSED** [review]
LGTM!

**Review by kcberg — APPROVED** [review]

**Pull request merged**